conans/client/cache/cache.py

```diff
@@ -27,6 +27,7 @@
 REMOTES = "remotes.json"
 PROFILES_FOLDER = "profiles"
 HOOKS_FOLDER = "hooks"
+SCHED_FILE = 'sched'
```

Comment:
maybe this file should start with dot? it's not something intended to be edited by user, right? also, it will avoid accidentally checking it into version control

Reply:
indeed, dot as hidden file sounds better.

Reply:
done